src/solver/NSTempTurbSolver.cpp

```diff
@@ -9,53 +9,51 @@
 #include "../boundary/BoundaryData.h"
 #include "../boundary/BoundaryController.h"
 #include "../pressure/VCycleMG.h"
-#include "../utility/Parameters.h"
 #include "SolverSelection.h"
 
-NSTempTurbSolver::NSTempTurbSolver(FieldController *field_controller) {
+NSTempTurbSolver::NSTempTurbSolver(Settings const &settings, FieldController *field_controller) :
+m_settings(settings) {
 #ifndef BENCHMARKING
-    m_logger = Utility::create_logger(typeid(this).name());
+    m_logger = Utility::create_logger(m_settings, typeid(this).name());
 #endif
     m_field_controller = field_controller;
 
-    auto params = Parameters::getInstance();
-
     // Advection of velocity
-    SolverSelection::SetAdvectionSolver(&adv_vel, params->get("solver/advection/type"));
+    SolverSelection::SetAdvectionSolver(m_settings, &adv_vel, m_settings.get("solver/advection/type"));
 
     // Advection of temperature
-    SolverSelection::SetAdvectionSolver(&adv_temp, params->get("solver/temperature/advection/type"));
+    SolverSelection::SetAdvectionSolver(m_settings, &adv_temp, m_settings.get("solver/temperature/advection/type"));
 
     // Diffusion of velocity
-    SolverSelection::SetDiffusionSolver(&dif_vel, params->get("solver/diffusion/type"));
+    SolverSelection::SetDiffusionSolver(m_settings, &dif_vel, m_settings.get("solver/diffusion/type"));
 
-    m_nu = params->get_real("physical_parameters/nu");
+    m_nu = m_settings.get_real("physical_parameters/nu");
```

Comment:
remove member variable `m_nu` instead access value through Settings

Reply:
done in 0d1d248